models/infrastructure_edge.py

```diff
@@ -2540,7 +2636,7 @@ async def run(
         resources=[supernet_prefix],
         branch=branch,
     )
-    await supernet_pool.save()
+    await supernet_pool.save(allow_upsert=True)
```

Comment:
Use upsert for branch agnostic nodes in order to execute the script in different branches during development

Reply:
This could be added as a comment